pkgs/development/ocaml-modules/syslog-message/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, buildDunePackage, fetchurl
+, astring, ptime, rresult, qcheck
+}:
+
+buildDunePackage rec {
+  pname = "syslog-message";
+  version = "1.1.0";
+
+  minimumOCamlVersion = "4.03";
```

Comment:
What’s the purpose of this line? The package fails to evaluate for OCaml < 4.04.

Reply:
copied from opam file, can remove.